app/models/house.rb

```diff
@@ -0,0 +1,3 @@
+class House < ActiveRecord::Base
+  has_many :student
```

Comment:
Should have been `has_many :students` plural.
This affects any time you use the relation, so `@house.student` would also need to become `@house.students`